layouts/partials/cookie_consent.html

```diff
@@ -0,0 +1,5 @@
+<script type="text/javascript">
+    window.cookieconsent_options = {{ dict "message" "This website uses cookies." "dismiss" "Got it!" "learnMore" "More info" "link" (.Site.Params.cookie_consent_info_url | absURL)  "theme" "dark-top" | jsonify | safeJS }};
+</script>
+
+<script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/1.0.10/cookieconsent.min.js"></script>
```

Comment:
Use https:// instead of //, and it looks like you're using an old version of cookieconsent2 ([CDNJS shows version 3.02](https://cdnjs.com/libraries/cookieconsent2)).